feat: upgrade latest version of dw-button(v2.2.0) and dw-icon-button(v2.4.0)

harshnasanghani opened the pull request.

harshnasanghani committed:
feat: START upgrade latest version of dw-button and dw-icon-button

harshnasanghani committed:
fix: circleci config.yml

hitendreamworld commented on yarn.lock:
```diff
@@ -91,34 +102,61 @@
   integrity sha512-HWd0+GMnkVB3anmUc9+MeWCNoogAbb4U7ihzwq7lzLCQyC+i/kunppkUVV7DhL3ZVl1O6zIw1eAT+dgQpN8x4Q==
 
-"@material/mwc-base@0.20.0", "@material/mwc-base@^0.20.0":
+"@material/mwc-base@0.20.0":
```
mwc-base has two entries. fix this and use latest version of it.

hitendreamworld committed:
fix: update snackbar render method

- used Array.map prototype instead of repeat directives
- update dw-button and dw-icon-button

dw-devops commented:
:tada: This PR is included in version 3.3.0 :tada:

The release is available on:
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/@dreamworld/dw-snackbar/v/3.3.0)
- [GitHub release](https://github.com/DreamworldSolutions/dw-snackbar/releases/tag/v3.3.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
